Client.xcodeproj/project.pbxproj

```diff
@@ -4488,6 +4488,9 @@
 							com.apple.Push = {
 								enabled = 1;
 							};
+							com.apple.SafariKeychain = {
+								enabled = 1;
+							};
```

Comment:
What is this `SafariKeychain` ? I have never seen this before .. got a link to documentation?